internal/protocoltest/awsrestjson/types/enums.go

```diff
@@ -6,11 +6,11 @@ type FooEnum string
 
 // Enum values for FooEnum
 const (
-	FooEnumFoo FooEnum = "Foo"
-	FooEnumBaz FooEnum = "Baz"
-	FooEnumBar FooEnum = "Bar"
-	FooEnum1   FooEnum = "1"
-	FooEnum0   FooEnum = "0"
+	FooEnumFoo  FooEnum = "Foo"
+	FooEnumBaz  FooEnum = "Baz"
+	FooEnumBar  FooEnum = "Bar"
+	FooEnumOne  FooEnum = "1"
+	FooEnumZero FooEnum = "0"
 )
```

Comment:
Guessing this is a upstream smithy-go change that didn't make it in before?

Reply:
This probably is due to having to use the tip of smithy to incorporate fixes for pagination token paths.